confdb/confdb.go

```diff
@@ -634,10 +673,20 @@ type Accessor interface {
 
 	// Type returns a type that represents the kind of path sub-key the accessor is.
 	Type() AccessorType
+
+	// FieldFilters returns a mapping of field names to filters to which
+	// constraints may be applied.
+	FieldFilters() map[string]string
+}
+
+type splitSubkey struct {
+	value   string
```

Comment:
value is a odd name for something that is part of a key,  maybe part or k or something like that